gateway/src/api/latest.rs

```diff
@@ -209,23 +209,35 @@ async fn create_project(
     CustomErrorPath(project_name): CustomErrorPath<ProjectName>,
     AxumJson(config): AxumJson<project::Config>,
 ) -> Result<AxumJson<project::Response>, Error> {
-    let cch_modifier = project_name.starts_with("cch23-");
+    let is_cch_project = project_name.is_cch_project();
 
     // Check that the user is within their project limits.
     let can_create_project = claim.can_create_project(
         service
             .get_project_count(&name)
             .await?
-            .saturating_sub(cch_modifier as u32),
+            .saturating_sub(is_cch_project as u32),
     );
 
+    if is_cch_project {
+        let current_container_count = service.count_ready_projects().await?;
```

Comment:
Should we trace the count so we can easily see in honeycomb how many projects are ready?

Reply:
Hmm, we don't run this query every time. Only for cch projects (in this PR at least). So the data in honeycomb will be quite limiting...

Reply:
It's been added to the ambulance trace